proto/ibc/applications/interchain_accounts/v1/types.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package ibc.applications.interchain_accounts.v1;
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types";
+
+// Raw tx body
+message IBCTxRaw {
```

Comment:
What is this used for?

Reply:
We are using this to unmarshal data (SDK messages) sent in the IBC transaction -> https://github.com/cosmos/ibc-go/blob/interchain-accounts/modules/apps/27-interchain-accounts/keeper/relay.go#L97

See here also -> https://github.com/cosmos/ibc-go/blob/interchain-accounts/modules/apps/27-interchain-accounts/keeper/relay.go#L210

Reply:
It just seems like an unnecessary wrapper. Don't we just need one type with an array of Any's. Then we marshal/unmarshal that and pass the marshaled version as `[]byte` in the packet?

Reply:
You are correct. I think this is left over from when there was a concept of having different types of deserialization for different types of chains. Seeing as we are only focusing on SDK chains for now we can probably remove it. I'll do it in a follow-up PR. 

I'll make a few issues out of the discussions here for leftover work.